### ISSUE TITLE
Development mode requires a cert with an OCSP URL

### ISSUE THREAD

**Issue by twifkak** [ampproject/amppackager#352]
Using the [instructions](https://github.com/WICG/webpackage/tree/master/go/signedexchange) to generate a self-signed cert, this doesn't work with `amppkg -development`:

```
2019/10/09 07:33:59 Updating OCSP; none cached yet.
2019/10/09 07:33:59 Updating OCSP; none cached yet.
2019/10/09 07:33:59 Error computing OCSP midpoint: Parsing OCSP: no response matching the supplied certificate
2019/10/09 07:33:59 Error computing OCSP midpoint: Parsing OCSP: no response matching the supplied certificate
2019/10/09 07:33:59 Error extracting OCSP server: Cert missing OCSPServer.
2019/10/09 07:33:59 Error parsing OCSP response: no response matching the supplied certificate
2019/10/09 07:33:59 OCSP failed health check.
```

We should fix `amppkg -development` not to require an OCSP URL, and in the meantime, document how to generate a self-signed cert with a fake AIA extension.

**Comment by twifkak** [comment]
[testdata/b1](https://github.com/ampproject/amppackager/tree/releases/testdata/b1) documents how to get around this, by pre-seeding the OCSP cache with a self-signed OCSP response. (But we should make that not necessary.)

**Comment by twifkak** [comment]
This is still not fixed. ServeHTTP calls readOCSP which still errors. I guess we should pass the development bool into certcache, and if development and the cert doesn't have an OCSP URL, then generate a fake OCSP response.

In the meantime, one might be able to use `openssl ocsp` to forge a fake OCSP response and save it to the OCSPCache location.

**Comment by twifkak** [comment]
Note that the fix is in #360, pending I make some reviewer-requested changes & get it approved, but can be patched in the interim.
